google/cloud/aiplatform/featurestore/feature.py

```diff
@@ -461,3 +461,141 @@ def search(
+            entity_type_name (str):
+                Required. A fully-qualified entityType resource name or an entity_type ID to create Feature in
+                Example: "projects/123/locations/us-central1/featurestores/my_featurestore_id/entityTypes/my_entity_type_id"
+                or "my_entity_type_id" when project and location are initialized or passed, with featurestore_id passed.
+            featurestore_id (str):
+                Optional. Featurestore to create Feature in.
+            description (str):
+                Optional. Description of the Feature.
+            labels (Dict[str, str]):
+                Optional. The labels with user-defined
+                metadata to organize your Features.
+                Label keys and values can be no longer than 64
+                characters (Unicode codepoints), can only
+                contain lowercase letters, numeric characters,
+                underscores and dashes. International characters
+                are allowed.
+                See https://goo.gl/xmQnxf for more information
+                on and examples of labels. No more than 64 user
+                labels can be associated with one Feature
+                (System labels are excluded)."
+                System reserved label keys are prefixed with
+                "aiplatform.googleapis.com/" and are immutable.
+            project (str):
+                Optional. Project to create Feature in. If not set, project
+                set in aiplatform.init will be used.
+            location (str):
+                Optional. Location to create Feature in. If not set, location
+                set in aiplatform.init will be used.
+            credentials (auth_credentials.Credentials):
+                Optional. Custom credentials to use to create Features. Overrides
+                credentials set in aiplatform.init.
+            request_metadata (Sequence[Tuple[str, str]]):
+                Optional. Strings which should be sent along with the request as metadata.
+            sync (bool):
+                Optional. Whether to execute this creation synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+
+        Returns:
+            Feature - feature resource object
+
+        """
+        (
+            featurestore_id,
+            entity_type_id,
+        ) = featurestore_utils.validate_and_get_entity_type_resource_ids(
+            entity_type_name=entity_type_name, featurestore_id=featurestore_id,
+        )
+
+        # TODO(b/208269923): Temporary workaround, update when base class supports nested resource
+        cls._resource_noun = (
+            f"featurestores/{featurestore_id}/entityTypes/{entity_type_id}/features"
+        )
+
+        entity_type_name = utils.full_resource_name(
```

Comment:
What is the behavior of this function if entity_type_name is fully-qualified but does not consistent with project and location or resource noun?

Reply:
If the `resource_name` arg for `utils.full_resource_name` is identified as a valid fully-qualified resource name, the arg value will be the return value of the method, and the overwrite `project` and `location` args will be ignored. 

The `resource_noun` is optional and only needed when the the `resource_name` arg value is identified as an invalid fully-qualified resource name, then a project and a location plus the `resource_noun` arg will be required to form the fully-qualified resource name. If `project` and `location` args can still be optional, if `project` and `location` set through `aiplatform.init`.